src/test/java/com/wikia/webdriver/testcases/portableinfoboxtests/PortableInfoboxTests.java

```diff
@@ -130,83 +133,70 @@ public void verifyCategoriesInTemplateInvocation() {
     info.verifyCategoryInArticlePage(catName);
   }
 
-  @Test(groups = {"PortableInfoboxTests", "PortableInfoboxTests_006"})
+  @Test(groups = "PortableInfoboxTests_006")
   public void verifyLightboxVisibilityAfterClickingImage() {
-    ArticlePageObject article = new ArticlePageObject(driver);
-    article.open(PageContent.PORTABLE_INFOBOX01);
-    PortableInfoboxPageObject info = article.getInfoboxPage();
-    info.clickImage();
-    info.verifyLightboxPresence();
+    new PortableInfoboxPageObject(driver)
+        .open(PageContent.PORTABLE_INFOBOX01)
+        .clickImage()
+        .verifyLightboxPresence();
   }
 
-  @Test(groups = {"PortableInfoboxTests", "PortableInfoboxTests_008"})
+  @Test(groups = "PortableInfoboxTests_008")
   public void verifyVisibilityOfTabberAndItsImages() {
-    ArticlePageObject article = new ArticlePageObject(driver);
-    article.open(PageContent.PORTABLE_INFOBOX02);
-    PortableInfoboxPageObject info = article.getInfoboxPage();
-    info.verifyTabberPresence();
-    info.verifyTabberImagePresence();
+    new PortableInfoboxPageObject(driver)
+        .open(PageContent.PORTABLE_INFOBOX02)
+        .verifyTabberPresence()
+        .verifyTabberImagePresence();
   }
 
+  // TODO: compress this test
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)